### DISCUSSION TITLE
Request: Reintroduce Google Drive View-Only Video Download Capability 🙏🙏🙏

### DISCUSSION THREAD

**Discussion by stas-sazhaev** [aclap-dev/video-downloadhelper#414] [Help and Q&A]
### Your question

My original reason for purchasing your extension was its ability to download videos that were set to view-only mode on Google Drive. I found this feature to be extremely useful and improved my user experience considerably. 🔥

Unfortunately, following recent updates from Google, it appears that your extension no longer detects videos from Google Drive. 😭 This change has markedly decreased the utility of the extension for me, and I suspect for other users as well. 

In light of how valuable this feature was, I would like to suggest that you consider reintroducing it into the extension. This capability would not only meet the needs of your existing users, but also enhance the appeal of your product to potential new users.

Thank you for considering this request. I am looking forward to your positive response and hoping for a swift resolution to this matter. 🙏🙏🙏

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: accepted
key (only 16 first characters): f2baf3b9d7d04776
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/stanislav",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by EloyFarina** [comment]
Same, for some reason this functionality is no longer.  But it worked years ago.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

I use Google Drive sparingly.  To the best of my knowledge, there is a download tool built into the Google Drive interface.  VDH is not needed.  Use the built in tool.

As the original poster indicated, Google has changed things.  When you're a monopoly, you get to do such things unilaterally, not subject to review or approval by outside agencies.  An objective observer might call such changes frivolous, unnecessary, devoid of added value.  But again, I draw your attention to their monopoly status.  You might occasionally manage to drag them into court on anti-competitive grounds.  But this would be rare.  Even if you won a legal decision against them, their compliance with that would probably lean more towards their convenience than yours.  Your victory would be pale.

So my advice is to forget about VDH on Google Drive & use the Google tool for downloading.  It works.  I've used it.  It works adequately, to the point that any such support in VDH seems entirely unnecessary to me.  I would vote for Paul to expend his effort on other more useful work.

**Reply by stas-sazhaev** [reply]
Hello! Thank you for taking the time to share your insights. In my previous message, I specifically referred to view-only mode videos. Unfortunately, the download function is not available in the Google Drive interface for such videos. This is where VDH came in handy before. I purchased a VDH license primarily because of this feature, and I’m eagerly awaiting its restoration.

**Reply by GrampaWildWilly** [reply]
Just a couple of days ago, I shared something on Google Drive with somebody & gave the other person view-only permission.  Assuming that's what you are talking about, I can't test what you're saying on that because I'm the owner of the content.  If you could share one of these view-only links on Google Drive, I could look at what happens.  I didn't realize the download button wasn't visible if the content was view-only.  Like I said, I have used Google Drive only occasionally & my memory is that the download button is always there.  But that could be a faulty memory so I want to look at something current.  So if it's not something of a very private nature, please share it.  You can always edit your post later, after I've had a chance to look at the thing, & delete the URL if you really want.

**Reply by GrampaWildWilly** [reply]
Scrap that.  I've added some commentary to #383.

**Comment by ZanzDroid** [comment]
I concur.  I also purchased a license for Video DownloadHelper, and would like to see the functionality restored to download View-Only files on Google Drive.

**Reply by TheBlinded** [reply]
Me too!! Any updates about this feature?
